### PR TITLE
[149] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by OluwaboriShobayo**
image
image

### CODE REVIEW

File: submissions/oluwaboriShobayo/html-forms/contactForm.html [hunk 1]
```diff
@@ -11,7 +11,7 @@ <h1>Style By Bori</h1>
         <a href="#">Categories</a><br>
         <a href="#">About us</a><br>
         <a href="../HTML/productPage.html">Products</a><br>
-        <a href="../html-forms/contactForm.html">Contact Us</a>
+        <a href="../ahtml-forms/contactForm.html">Contact Us</a>
```
Comment (Ireti203) [inline submissions/oluwaboriShobayo/html-forms/contactForm.html:14]: is it ahtml-forms or html-forms?

**Review by Ireti203 — CHANGES_REQUESTED** [review]
I cant see any navigation link on the productPage.

**Comment by OluwaboriShobayo** [comment]
> I cant see any navigation link on the productPage.

I don't understand what you mean please

**Comment by Ireti203** [comment]
> > I cant see any navigation link on the productPage.
> 
> I don't understand what you mean please

You should add navigation links to both pages. I can only see the links on contactForm.html

**Comment by OluwaboriShobayo** [comment]
> > > I cant see any navigation link on the productPage.
> > 
> > 
> > I don't understand what you mean please
> 
> You should add navigation links to both pages. I can only see the links on contactForm.html

I have checked again; The links are also on productPage.html from line 11 to 15

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**